cmd/root.go

```diff
@@ -113,12 +99,44 @@ func setupLogger(atmosConfig *schema.AtmosConfiguration) {
 		output = logFile
 	}
 
-	log.SetOutput(output)
+	atmosLogger := logger.NewAtmosLogger(output)
+
+	// Set the global log instance to our styled logger.
+	log.SetDefault(atmosLogger)
+
+	// Set the appropriate log level based on configuration.
+	switch atmosConfig.Logs.Level {
+	case "Trace":
+		atmosLogger.SetLevel(logger.AtmosTraceLevel)
+	case "Debug":
+		atmosLogger.SetLevel(log.DebugLevel)
+	case "Info":
+		atmosLogger.SetLevel(log.InfoLevel)
+	case "Warning":
+		atmosLogger.SetLevel(log.WarnLevel)
+	case "Error":
+		atmosLogger.SetLevel(log.ErrorLevel)
+	case "Off":
+		atmosLogger.SetLevel(math.MaxInt32)
+	default:
+		atmosLogger.SetLevel(log.InfoLevel)
+	}
+
 	if _, err := logger.ParseLogLevel(atmosConfig.Logs.Level); err != nil {
 		//nolint:all // The reason to escape this is because it is expected to fail fast. The reason for ignoring all is because we also get a lint error to execute defer logFile.Close() before this but that is not required
 		log.Fatal(err)
 	}
-	log.Debug("Set", "logs-level", log.GetLevel(), "logs-file", atmosConfig.Logs.File)
+	// Use the log level from config directly for consistent output
+	logLevelStr := strings.ToLower(atmosConfig.Logs.Level)
+	if logLevelStr == "" {
+		logLevelStr = "info" // Default value if not specified
+	}
+
+	if strings.ToLower(atmosConfig.Logs.Level) == "trace" {
+		log.Log(logger.AtmosTraceLevel, "Set", "logs-level", logLevelStr, "logs-file", atmosConfig.Logs.File)
+	} else {
+		log.Debug("Set", "logs-level", logLevelStr, "logs-file", atmosConfig.Logs.File)
```

Comment:
Here we use `log.Debug`, but in the logs we output `TRCE`

We should be consistent.

https://github.com/cloudposse/atmos/pull/1135/files#r2004173296

I'm okay changing it to `Trace`, but then we should have it be that consistently.